boards/m5stack/m5stack_cores3/m5stack_cores3_procpu_defconfig

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+
+
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_I2C=y
+CONFIG_GPIO=y
+#CONFIG_GPIO_AW9523B=y
+CONFIG_REGULATOR=y
+#CONFIG_REGULATOR_AXP2101=y
+
+CONFIG_SHELL=y
+CONFIG_GPIO_SHELL=y
+CONFIG_REGULATOR_SHELL=y
+
+CONFIG_LOG=y
+CONFIG_LOG_CMDS=y
+CONFIG_LOG_BACKEND_UART=y
+CONFIG_GPIO_LOG_LEVEL_DBG=y
+
```

Comment:
No

Reply:
> if not needed, get rid of

> No

temporarily use shell commands for debug regulator, gpio etc.

Reply:
removed.